libs/core/src/components/dropdown/dropdown.ts

```diff
@@ -293,7 +300,10 @@ export class GdsDropdown<ValueT = any>
           ${this.combobox && !this.multiple
             ? this.#renderCombobox()
             : this.#renderTriggerButton()}
-          <gds-icon-chevron-bottom slot="trail"></gds-icon-chevron-bottom>
+          <gds-icon-chevron-bottom
+            slot="trail"
+            label=${this.expandLabel}
```

Comment:
We can hard-code the icon label here instead, but make sure to make it localizable with `msg`:
```
label=${msg('Expand')}
```

Then you need to add the translations:
1. Enter the core directory: `$ cd libs/core`
2. Extract translation strings: `$ npx lit-localize extract`
3. Edit the language files in `xliff/*.xlf` and update the added entry for "Expand"
4. Generate updated typescript locale files: `$ npx lit-localize build`

Commit all the changes, including the generated files.

Read more here: https://lit.dev/docs/localization/overview/#extracting-messages